app/models/visit.rb

```diff
@@ -1,4 +1,23 @@
 class Visit < ActiveRecord::Base
-  belongs_to :links
-  # attr_accessor :link_id, :user_id, :ip_address
+  belongs_to :user
+  belongs_to :link
+
+  scope :popular_links, lambda {
```

Comment:
**Critical - Code Quality**
When working with multiple lines then make use of the `do..end` block as against `{ }`.
Hence, you'll have
```
scope :popular_links, lambda do 
...
end
```

Please fix all instances of this issue.